MultipleStringLiterals/Example3/config.xml

```diff
@@ -23,7 +23,7 @@
         <property name="javaParseExceptionSeverity" value="ignore"/>
 
         <module name="MultipleStringLiterals">
-            <property name="ignoreStringsRegexp" value="^(("")|(", "))$"/>
+            <property name="ignoreStringsRegexp" value='^(("")|(", "))$'/>
```

Comment:
https://checkstyle.org/checks/coding/multiplestringliterals.html#Example3-config

We have single quotes